nemo/collections/tts/models/magpietts_preference_optimization.py

```diff
@@ -435,6 +435,11 @@ def __init__(self, cfg: DictConfig, trainer: 'Trainer' = None):
             # import ipdb; ipdb.set_trace()
             assert HAVE_TORCHAUDIO, "torchaudio is required for PESQ reward"
             self.squim_objective_model = SQUIM_OBJECTIVE.get_model()
+        
+        self.loss_type = self.cfg.get('loss_type', 'grpo')
+        assert self.loss_type in ['grpo', 'dr_grpo'], f"Unknown loss type: {self.loss_type}"
```

Comment:
```suggestion
        if self.loss_type not in ['grpo', 'dr_grpo']:
            raise ValueError(f"Received loss_type of {self.loss_type}, but the model only accepts one of ['grpo', 'dr_grpo']")
```
Try to avoid assert statements, and raise Exceptions instead. If possible, mention how the user can fix this error as well.